tests/ses/openattic.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: openattic test
+# Maintainer: Jozef Pupava <jpupava@suse.cz>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use x11test;
+use mm_network;
+use lockapi;
+use utils 'zypper_call';
+
+sub run {
+    my ($self) = @_;
+    if (check_var('HOSTNAME', 'master')) {
+        # install firefox and icewm to test openattic
+        zypper_call 'in firefox icewm xinit xorg-x11-server';
```

Comment:
If you use `sle-12-SP3-Server-DVD-x86_64-gnome-CaaSP.qcow2` as your support server you can benefit from few "hacks" :)
No firefox popups (first use, save password, default browser) and gnome + firefox are already installed.

But it sounds like CaaSP commercial - leave it as it is if you prefer this way :)

Reply:
I wanted to do it fist via support_server, but I didn't know about this and did it like it is, but I guess I will change it ... later